iexec-sms-library/src/main/java/com/iexec/sms/api/SmsClient.java

```diff
@@ -90,14 +104,23 @@ ApiResponseBody<TeeSessionGenerationResponse, TeeSessionGenerationError> generat
             WorkerpoolAuthorization workerpoolAuthorization
     );
 
-    @RequestLine("GET /tee/workflow/config")
-    TeeWorkflowConfiguration getTeeWorkflowConfiguration();
+    @RequestLine("GET /tee/provider")
+    TeeEnclaveProvider getTeeEnclaveProvider();
 
-    @RequestLine("POST /untee/secrets")
-    @Headers("Authorization: {authorization}")
-    SmsSecretResponse getUnTeeSecrets(
-            @Param("authorization") String authorization,
-            WorkerpoolAuthorization workerpoolAuthorization
-    );
+    @RequestLine("GET /tee/config/scone")
+    SconeServicesProperties getSconeServicesConfiguration();
```

Comment:
```suggestion
    SconeServicesProperties getSconeServicesProperties();
```
`GET /tee/{scone}/properties`
?